src/logging.c

```diff
@@ -0,0 +1,155 @@
+/* ureadahead
+ *
+ * logging.c - logging implementation
+ *
+ * Copyright © 2009 Canonical Ltd.
+ * Author: Scott James Remnant <scott@netsplit.com>.
```

Comment:
I don't think this is correct. Could you update the license sections of all new files including this copy right with the right template which you can find via moma?